packages/amplify-category-geo/src/service-walkthroughs/resourceWalkthrough.ts

```diff
@@ -52,4 +81,9 @@ const getServiceFriendlyName = (service: ServiceName): string => {
         default:
             return service;
     }
-};
\ No newline at end of file
+};
+
+export const enum YesOrNo {
```

Comment:
I don't think this needs to be an enum. If you want a type for it, use
```
export type YesOrNo = 'yes' | 'no';
```
but it seems like just using a boolean would work as well

Reply:
removed this enum.